server/parsec/components/postgresql/__init__.py

```diff
@@ -1,6 +1,11 @@
 # Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
 from __future__ import annotations
 
+import asyncpg
+
+type PgConn = asyncpg.Connection[asyncpg.Record]
+type PgPool = asyncpg.Pool[asyncpg.Record]
```

Comment:
Can it be interesting to add module name to type name? 
I'm asking because our components are also prefixed with `PG...` (i.e. `PGUserComponent`) 
 
```suggestion
type AsyncpgConnection = asyncpg.Connection[asyncpg.Record]
type AsyncpgPool = asyncpg.Pool[asyncpg.Record]
```

Reply:
I like that :+1: